{\rtf1\ansi\ansicpg1252\cocoartf2867
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
{\*\expandedcolortbl;;}
\paperw11900\paperh16840\margl1440\margr1440\vieww29200\viewh15480\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0

\f0\fs24 \cf0 diff --git a/frontend/app.json b/frontend/app.json\
index 0fa96f1..27f5ea6 100644\
--- a/frontend/app.json\
+++ b/frontend/app.json\
@@ -33,7 +33,8 @@\
           "resizeMode": "contain",\
           "backgroundColor": "#000"\
         \}\
-      ]\
+      ],\
+      "react-native-reanimated/plugin"\
     ],\
     "experiments": \{\
       "typedRoutes": true\
diff --git a/frontend/babel.config.js b/frontend/babel.config.js\
new file mode 100644\
index 0000000..fbaf1f9\
--- /dev/null\
+++ b/frontend/babel.config.js\
@@ -0,0 +1,9 @@\
+module.exports = function (api) \{\
+  api.cache(true);\
+  return \{\
+    presets: ['babel-preset-expo'],\
+    plugins: [\
+      'react-native-reanimated/plugin',\
+    ],\
+  \};\
+\};\
diff --git a/frontend/package.json b/frontend/package.json\
index 9b37966..2ce8291 100644\
--- a/frontend/package.json\
+++ b/frontend/package.json\
@@ -11,6 +11,7 @@\
     "lint": "expo lint"\
   \},\
   "dependencies": \{\
+    "@babel/runtime": "^7.20.6",\
     "@expo/ngrok": "^4.1.3",\
     "@expo/vector-icons": "^14.1.0",\
     "@react-native-async-storage/async-storage": "^2.2.0",\
@@ -33,16 +34,17 @@\
     "expo-system-ui": "~5.0.10",\
     "expo-web-browser": "~14.2.0",\
     "lottie-react-native": "^7.3.4",\
-    "react": "19.0.0",\
-    "react-dom": "19.0.0",\
-    "react-native": "0.79.5",\
+    "react": "19.1.0",\
+    "react-dom": "19.1.0",\
+    "react-native": "0.81.4",\
     "react-native-dotenv": "^3.4.11",\
     "react-native-gesture-handler": "~2.24.0",\
-    "react-native-reanimated": "~3.17.4",\
+    "react-native-reanimated": "~4.0.0",\
     "react-native-safe-area-context": "5.4.0",\
     "react-native-screens": "~4.11.1",\
     "react-native-web": "~0.20.0",\
-    "react-native-webview": "13.13.5"\
+    "react-native-webview": "13.13.5",\
+    "react-native-worklets": "^0.4.0"\
   \},\
   "devDependencies": \{\
     "@babel/core": "^7.25.2",}